pkg/koordlet/statesinformer/impl/states_pods.go

```diff
@@ -155,10 +223,15 @@ func (s *podsInformer) syncPods() error {
 	for i := range podList.Items {
 		pod := &podList.Items[i]
 		podMeta := &statesinformer.PodMeta{
-			Pod:       pod, // no need to deep-copy from unmarshalled
-			CgroupDir: genPodCgroupParentDir(pod),
+			Pod:              pod, // no need to deep-copy from unmarshalled
+			CgroupDir:        genPodCgroupParentDir(pod),
+			ContainerTaskIds: make(map[string][]int32),
```

Comment:
perhaps postpone the initialization inside the `if s.config.EnablePodTaskIds` or `getTaskIds`

Reply:
yes, we can postpone this.